front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -121,14 +123,78 @@ export function AgentMessage({
   const agentMessageToRender =
     message.status === "succeeded" ? message : streamedAgentMessage;
 
+  if (agentMessageToRender.visibility === "deleted") {
```

Comment:
I assumed this was the desired behaviour for visibility = deleted, correct me if wrong

Reply:
Won't that create spacing in the conversation?